tests/src/test/scala/cats/tests/OneAndTests.scala

```diff
@@ -13,6 +13,8 @@ import scala.util.Random
 class OneAndTests extends CatsSuite {
   checkAll("OneAnd[List, Int]", OrderLaws[OneAnd[List, Int]].eqv)
 
+  checkAll("OneAnd[List, Int] with Option", TraverseTests[OneAnd[List, ?]].traverse[Int, Int, Int, Int, Option, Option])
```

Comment:
Can you also add the check the serializability of the `Traverse` instance as well?